### ISSUE TITLE
ci: support for triggering long tests

### ISSUE THREAD

**Issue by myitcv** [cue-lang/cue#1876]
At the moment we can't trigger the long tests on via our trybots. 

We should add the ability to do so via [`cmd/cueckoo`](https://github.com/cue-sh/tools/tree/master/cmd/cueckoo).

**Comment by mvdan** [comment]
`CUE_LONG=1` disappeared as of https://review.gerrithub.io/c/cue-lang/cue/+/1192938, so I think this is no longer needed. At least not until we need to re-introduce "long" or "slow" tests, and hopefully that doesn't happen anytime soon.
